anomalib/data/base/video.py

```diff
@@ -93,3 +94,25 @@ def __getitem__(self, index: int) -> Dict[str, Union[str, Tensor]]:
             item.pop("mask")
 
         return item
+
+
+class VideoAnomalibDataModule(AnomalibDataModule):
```

Comment:
Which one do you guys think would sound more natural `VideoAnomalibDataModule ` or `AnomalibVideoDataModule`?

Reply:
Yeah I also noticed this earlier. I guess AnomalibVideoDataset would be better. But since it's not related to this PR, let's address it separately.

Reply:
nvm, it was a small effort, so I added it to this one to speed things up